### DISCUSSION TITLE
Kakute F7 v3 mini problem with Compass-GPS Beitian 880

### DISCUSSION THREAD

**Discussion by skimans** [iNavFlight/inav#6551] [Q&A]
Hi have problem with fc Kakute F7 mini v3 using GPS Beitian 880.
The problem is that the RTH not works fine.
My quad go to another direction and not to RTH.
I think can be interference on flight controller but i'm not sure.
I dont know if changed firmware is posible fix this issue.
how i can resolve ?
thanks

**Comment by avsaase** [comment]
I am using the same fc and gps/compass and I do not experience any problems. I would check magnetometer alignment and make sure it is no too close to other electronics.
